common/service-support-cdi/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
`common` module should only contain modules that are Helidon SE.
MP modules should not exist here. If you want, you can create a `microprofile/common` module to add these classes to, or make them part of `helidon-microprofile-cdi` (once you resolve my other comments related to server dependency).

Reply:
Addressed by creating the new top-level module (named according to vote) and submodules as described here: https://gist.github.com/tjquinno/d5b5d13d4e1220e6631372c59d2f7d8b